### PR TITLE
Gallery workflow

### PR DESCRIPTION
**Pull request by myh-0521**
---
name: Pull Request
about: Create a pull request
---
fix bug  and  gallery refine

[Please describe the background, purpose, changes made, and how to test this PR]

## Checklist

Please check the following items before code is ready to be reviewed.

- [ ]  Code has passed all tests
- [ ]  Docstrings have been added/updated in Google Style
- [ ]  Documentation has been updated
- [ ]  Code is ready for review

### CODE REVIEW

**Comment by rayrayraykk** [comment]
请描述一下这个PR修复了什么问题

**Comment by myh-0521** [comment]
Bug：缩小拖拽出来的节点的时候，显示有bug，Bug: 拖拽界面可以调整每个节点的大小，但是这个信息目前没有包含在导出的config里，因而导入之后，很可能会出现某个节点不在另一个group节点之中的情况

File: src/agentscope/studio/static/js/gallery.js [hunk 1]
```diff
@@ -248,14 +261,11 @@ function showLoadWorkflowList(tabId) {
...
       const container = document.getElementById(tabId).querySelector(".grid-container");
       container.innerHTML = "";
-
-      data.files.forEach(workflowName => {
-        const title = workflowName.replace(/\.json$/, "");
-        const thumbnail = generateThumbnailFromContent({ title });
-        createGridItem(title, container, thumbnail, "", "", true);
+      data.files.forEach(fileName => {
+        const thumbnail = generateThumbnailFromContent({title: fileName});
+        createGridItem(fileName, container, thumbnail, "", "", true); // 传递完整的文件名
```
Comment (qbc2016) [inline src/agentscope/studio/static/js/gallery.js:268]: please delete the comment or translate it to English (same for the comments below)

File: src/agentscope/studio/static/js/workstation.js [hunk 2]
```diff
@@ -2366,57 +2363,45 @@ async function fetchHtmlSourceCodeByName(name) {
 
 async function addHtmlAndReplacePlaceHolderBeforeImport(data) {
   const idPlaceholderRegex = /ID_PLACEHOLDER/g;
-  const namePlaceholderRegex = /NAME_PLACEHOLDER/g;
```
Comment (qbc2016) [inline src/agentscope/studio/static/js/workstation.js:2378]: why delete these

**Pull request closed without merging**